api/rest-client.ts

```diff
@@ -1,9 +1,24 @@
-import axios, { AxiosRequestConfig, ResponseType } from 'axios'
+import https from 'https'
+import got, { NormalizedOptions, Options, Headers } from 'got'
+import CacheableLookup from 'cacheable-lookup'
 import { delay, getHardwareId, logError, logInfo, stringify } from './util'
 import * as querystring from 'querystring'
 import { AuthTokenResponse, SessionResponse } from './ring-types'
 import { ReplaySubject } from 'rxjs'
 
+const cacheableLookup = new CacheableLookup();
+const httpsAgent = new https.Agent({ keepAlive: true });
+https.globalAgent = httpsAgent;
+
+const defaultOptions : Options = {
+  headers: { 'user-agent': 'Mozilla/5.0 (Linux) Ring' },
```

Comment:
Not sure where this user agent came from...

Reply:
Was trying to resolve a 400 Bad Request error, probably didn't help at all. Shouldn't we add something though? Like plain "Device", as I read some of the communication excerpts with the Ring servers.

Reply:
Not a big deal either way.  They aren't using `user-agent` to do any verification on their end, so no need to provide it.